samples/wifi/sta/README.rst

```diff
@@ -23,6 +23,11 @@ This sample can perform Wi-Fi operations such as connect and disconnect in the 2
 
 Using this sample, the development kit can connect to the specified access point in :abbr:`STA (Station)` mode.
 
+Power management
+****************
+This sample also enables Zephyr's power management policy by default, which puts the nRF5340 :term:`System on Chip (SoC)` into low power mode whenever it is idle.
+See :ref:`zephyr:pm` in the Zephyr documentation for more information on power management.
```

Comment:
I think there is some problem in "Reference" which is leading to unsuccessful doc build. You can try this if you want to link to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/services/pm/index.html page.
```suggestion
See :ref:`zephyr:pm-guide` in the Zephyr documentation for more information on power management.
```